v2/src/log_context_property_type_wchar_t_ptr.c

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <wchar.h>
+#include <corecrt_wstdio.h>
```

Comment:
did this even compile in Linux? Looks like Windows specific file.

Reply:
doesn't compile on linux, deleted